### ISSUE TITLE
I am having problem on using the api myself can you help?

### ISSUE THREAD

**Issue by ArjixWasTaken** [huygtu/hanime-comments#1]
So I basically want to use the api to get the images that are uploaded on the website
the api url is https://hr.hanime.tv/api/v8/community_uploads
but no matter what i always get an unauthorized access error
can you help me?
my discord is Arjix#7453

PS: full url to the link with the json response i want to get is: 
https://hr.hanime.tv/api/v8/community_uploads?channel_name__in[]=nsfw-general&query_method=offset&__offset=0&loc=https://hanime.tv

**Comment by ArjixWasTaken** [comment]
This is not an issue with the repo, it's more like a request

**Comment by whew** [comment]
The hanime API is undocumented and it doesn't look like any of the independent documentation efforts (e.g. [1s3k3b](https://github.com/1s3k3b/hanime-api), [sk-1982](https://github.com/sk-1982/hanime)) have covered this endpoint of the API.

When I went to https://hanime.tv/browse/images my browser also sent a request to your API URL, and these are the request headers that it sent:

![hanime http headers](https://user-images.githubusercontent.com/73732390/100518416-41d9b780-3189-11eb-8037-83a1774ee86f.png)

Your requests need those authorization headers to get through. When I made this repo I was able to get away with random values for those headers in all my API calls. I tried that with your API URL but it doesn't work, always 401 Unauthorized.

AFAIK your only real option now is to automate a web browser to do the requests for you, e.g. with selenium.
<hr>

TL;DR: you have to use a web browser, sorry

**Comment by ArjixWasTaken** [comment]
Thanks for the response, no wonder I couldn't use that API endpoint, so I guess only selenium would do the job.
This "issue" is closed!
